randovania/games/prime1/logic_database/Tallon Overworld.txt

```diff
@@ -260,7 +277,29 @@ Extra - aabb: [-579.22754, 296.85858, -35.60154, -454.3111, 409.0403, 42.921246]
   > Door to Transport Tunnel A
       Trivial
   > Door to Gully
-      Morph Ball Bomb and Boost Ball and Morph Ball
+      All of the following:
+          Any of the following:
+              # Way to top of half-pipe
+              Boost Ball and Morph Ball
+              All of the following:
+                  Space Jump Boots
+                  Any of the following:
+                      # https://youtu.be/lN6aTh05TgA
+                      Standable Terrain (Intermediate)
+                      # https://youtu.be/K2hpJEYeIeg
+                      Combat/Scan Dash (Intermediate)
+                      # https://youtu.be/X-pKDXxZh6s
+                      Morph Ball Bomb and Morph Ball and Bomb Space Jump (Beginner)
+                      # https://youtu.be/lN6aTh05TgA
+                      R-Jump (Beginner) and Slope Jump (Intermediate)
+          All of the following:
+              # Method to get past blocks
+              Morph Ball
+              Any of the following:
+                  Morph Ball Bomb
+                  Power Bomb and Knowledge (Beginner)
+                  # https://youtu.be/IHV-S_1WM9s
+                  Boost Ball and Space Jump Boots and Single Room Out of Bounds (Intermediate) and R-Jump (Intermediate) and Standable Terrain (Intermediate)
```

Comment:
You'll need `Clip Through Objects` for this trick. I'd probably say Advanced is good.

Reply:
advanced? it's not too bad, and I never boosted back OoB in attempts. fine with either int or advanced but wanted to discuss

Reply:
I guess transport access OoB is adv, so fine with that